EventFilter/HGCalRawToDigi/src/HGCalUnpacker.cc

```diff
@@ -51,10 +52,25 @@ uint16_t HGCalUnpacker::parseFEDData(unsigned fedId,
   // Others: big endianness
   const auto* const header = reinterpret_cast<const uint64_t*>(fed_data.data());
   const auto* const trailer = reinterpret_cast<const uint64_t*>(fed_data.data() + fed_data.size());
+#ifdef EDM_ML_DEBUG
   LogDebug("[HGCalUnpacker]") << "fedId = " << fedId << ", nwords (64b) = " << std::distance(header, trailer);
+#endif
```

Comment:
Enclosing `LogDebug` inside `#ifdef EDM_ML_DEBUG` is redundant, because `LogDebug` internally compiles to no-op if `EDM_ML_DEBUG` is not defined (see e.g. https://github.com/cms-sw/cmssw/blob/master/FWCore/MessageService/Readme.md#have-all-debug-messages-be-shown)